modules/wordads/wordads.php

```diff
@@ -365,26 +395,31 @@ function get_ad( $spot, $type = 'iponweb' ) {
 
 	/**
 	 * Returns the snippet to be inserted into the ad unit
-	 * @param  int $section_id
 	 * @param  int $height
 	 * @param  int $width
+	 * @param  int $location
 	 * @param  string $css
 	 * @return string
 	 *
 	 * @since 5.7
 	 */
-	function get_ad_snippet( $section_id, $height, $width, $css = '' ) {
-		$this->ads[] = array( 'id' => $section_id, 'width' => $width, 'height' => $height );
+	function get_ad_snippet( $height, $width, $location = '', $css = '' ) {
+		$this->ads[] = array( 'location' => $location, 'width' => $width, 'height' => $height );
+		$ad_number = count( $this->ads );
+		// Max 6 ads per page.
+		if ( $ad_number > 6 ) {
```

Comment:
While this will stop ads from being displayed, it will not stop them being pushed to the `ads` array. Is that the intention?

Reply:
@rclations can you reply here please ?

Reply:
I think this is fine - that array is only used for this purpose (hiding > 6 ads), and the array is intended to hold a record of all the ads we're attempting to display on the page.